github/with_appengine.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build appengine
+
+// This file provides glue for making github work on App Engine.
+// In order to get the entire github package to compile with
+// Go 1.6, you will need to rewrite all the import "context" lines.
+// Fortunately, this is easy with "gofmt":
+//
+//     gofmt -w -r '"context" -> "golang.org/x/net/context"' *.go
+
+package github
+
+import (
+	"net/http"
+
+	"golang.org/x/net/context"
```

Comment:
Since all other files import `"context"`, and the `gofmt -w -r '"context" -> "golang.org/x/net/context"' *.go` step is necessary, is there any advantage to importing `"golang.org/x/net/context"` here instead of `"context"`?

In the end, it doesn't matter, but I'm leaning slightly towards preferring `"context"` because it's more consistent. Either `"context"` package doesn't exist, and the rewrite is neccessary everywhere, or it does exist, and nothing needs to be done anywhere. It would avoid importing both types of context package if `"context"` happens to be (or becomes) available.

I don't feel too strongly about this, so feel free to go with whatever you think is best.